Gruntfile.js

```diff
@@ -10,24 +10,6 @@ module.exports = (grunt) => {
         },
       },
     },
-    copy: {
-      dev: {
-        files: [
-          {
-            expand: true,
-            flatten: true,
-            src: ['node_modules/font-awesome/fonts/*'],
-            dest: 'public/fonts',
-          },
-          {
-            expand: true,
-            cwd: 'src/indigo',
-            src: ['img/**'],
-            dest: 'public/',
-          },
-        ],
-      },
-    },
```

Comment:
Navíc promazáno, ty imgs nepoužíváme v kbc-ui nikde a ten font to je něco legacy..